crates/standalone/src/subcommands/start.rs

```diff
@@ -50,170 +56,146 @@ impl ProgramMode {
-            database_path_arg = database_path_arg.default_value(paths.db_path().into_os_string());
-            jwt_pub_key_path_arg = jwt_pub_key_path_arg.default_value(paths.public_key().into_os_string());
-            jwt_priv_key_path_arg = jwt_priv_key_path_arg.default_value(paths.private_key().into_os_string());
+            data_dir_arg = data_dir_arg.required(false);
         }
         ProgramMode::Standalone => {
-            let paths = FilesGlobal;
-
-            log_conf_path_arg = log_conf_path_arg.default_value(paths.log_config().into_os_string());
-            log_dir_path_arg = log_dir_path_arg.default_value(paths.logs().into_os_string());
-            database_path_arg = database_path_arg.default_value(paths.db_path().into_os_string());
-            jwt_pub_key_path_arg = jwt_pub_key_path_arg.default_value(paths.public_key().into_os_string());
-            jwt_priv_key_path_arg = jwt_priv_key_path_arg.default_value(paths.private_key().into_os_string());
+            data_dir_arg = data_dir_arg.required(true);
         }
     }
 
     clap::Command::new("start")
         .about("Starts a standalone SpacetimeDB instance")
-        .long_about("Starts a standalone SpacetimeDB instance. This command recognizes the following environment variables: \
-                \n\tSPACETIMEDB_LOG_CONFIG: The path to the log configuration file. \
-                \n\tSPACETIMEDB_LOGS_PATH: The path to the directory that should contain logs for SpacetimeDB. \
-                \n\tSTDB_PATH: The path to the directory that should contain the database files for SpacetimeDB. \
-                \n\tSPACETIMEDB_JWT_PUB_KEY: The path to the public jwt key for verifying identities. \
-                \n\tSPACETIMEDB_JWT_PRIV_KEY: The path to the private jwt key for issuing identities. \
-                \n\tSPACETIMEDB_TRACY: Set to 1 to enable Tracy profiling.\
-                \n\nWarning: If you set a value on the command line, it will override the value set in the environment variable.")
+        // .long_about("Starts a standalone SpacetimeDB instance. This command recognizes the following environment variables: \
+        //         \n\tSPACETIMEDB_LOG_CONFIG: The path to the log configuration file. \
+        //         \n\tSPACETIMEDB_LOGS_PATH: The path to the directory that should contain logs for SpacetimeDB. \
+        //         \n\tSTDB_PATH: The path to the directory that should contain the database files for SpacetimeDB. \
+        //         \n\tSPACETIMEDB_JWT_PUB_KEY: The path to the public jwt key for verifying identities. \
+        //         \n\tSPACETIMEDB_JWT_PRIV_KEY: The path to the private jwt key for issuing identities. \
+        //         \n\tSPACETIMEDB_TRACY: Set to 1 to enable Tracy profiling.\
+        //         \n\nWarning: If you set a value on the command line, it will override the value set in the environment variable.")
```

Comment:
Those are the old env vars that we don't read anymore - I figured it might be nice to leave for future reference, but I guess that's what commit history is for.